addok/batch.py

```diff
@@ -15,11 +17,24 @@ def run(args):
         process_stdin(sys.stdin)
 
 
+def flush(args):
```

Comment:
In Redis world, `flushall` means "flush all db", so I think it could be confusing.